app/controllers/AccountManagement.scala

```diff
@@ -353,12 +354,13 @@ object AccountManagement extends Controller with ContextLogging with CatalogProv
 
     val futureMaybeFutureManagePage = for {
       subscription <- OptionT(eventualMaybeSubscription).filter(!_.isCancelled)
+      account <- OptionT(tpBackend.zuoraService.getAccount(subscription.accountId).map{Some(_)}.recover{case t: Throwable => None})
```

Comment:
This call used to happen inside [billingSchedule](https://github.com/guardian/membership-common/blob/e9e2118695c7c5d52463b8a7300a339795bdf959/src/main/scala/com/gu/memsub/services/PaymentService.scala#L91)